### PR TITLE
[node-manager] Clarify using NodeGroup parameters

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Clarify using NodeGroup parameters.

## Changelog entries
```changes
section: docs
type: chore
summary: Clarify using NodeGroup parameters.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -350,21 +350,23 @@ spec:
 
 | Параметр NG                           | Disruption update          | Перезаказ узлов   | Рестарт kubelet |
 |---------------------------------------|----------------------------|-------------------|-----------------|
-| kubelet.maxPods                       | -                          | -                 | +               |
-| kubelet.rootDir                       | -                          | -                 | +               |
+| chaos                                 | -                          | -                 | -               |
+| cloudInstances.classReference         | -                          | +                 | -               |
+| cloudInstances.maxSurgePerZone        | -                          | +                 | -               |
```
Comment (yalosev) [inline modules/040-node-manager/docs/FAQ_RU.md:355]:
```suggestion
| cloudInstances.maxSurgePerZone        | -                          | -                 | -               |
```

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 2]
```diff
@@ -350,21 +350,23 @@ spec:
...
-| chaos                                 | -                          | -                 | -               |
+| disruptions                           | -                          | -                 | -               |
+| kubelet.maxPods                       | -                          | -                 | +               |
+| kubelet.rootDir                       | -                          | -                 | +               |
 | kubernetesVersion                     | -                          | -                 | +               |
+| nodeTemplate                          | -                          | -                 | -               |
 | static                                | -                          | -                 | +               |
-| disruptions                           | -                          | -                 | -               |
-| cloudInstances.classReference         | -                          | +                 | -               |
+| update.maxConcurrent                  | -                          | +                 | -               |
```
Comment (yalosev) [inline modules/040-node-manager/docs/FAQ_RU.md:365]:
```suggestion
| update.maxConcurrent                  | -                          | -                 | -               |
```

File: modules/040-node-manager/docs/FAQ.md [hunk 3]
```diff
@@ -350,21 +350,23 @@ spec:
 
 | The NodeGroup parameter               | Disruption update          | Node provisioning | Kubelet restart |
 |---------------------------------------|----------------------------|-------------------|-----------------|
-| kubelet.maxPods                       | -                          | -                 | +               |
-| kubelet.rootDir                       | -                          | -                 | +               |
+| chaos                                 | -                          | -                 | -               |
+| cloudInstances.classReference         | -                          | +                 | -               |
+| cloudInstances.maxSurgePerZone        | -                          | +                 | -               |
```
Comment (yalosev) [inline modules/040-node-manager/docs/FAQ.md:355]:
```suggestion
| cloudInstances.maxSurgePerZone        | -                          | -                 | -               |
```

File: modules/040-node-manager/docs/FAQ.md [hunk 4]
```diff
@@ -350,21 +350,23 @@ spec:
...
-| chaos                                 | -                          | -                 | -               |
+| disruptions                           | -                          | -                 | -               |
+| kubelet.maxPods                       | -                          | -                 | +               |
+| kubelet.rootDir                       | -                          | -                 | +               |
 | kubernetesVersion                     | -                          | -                 | +               |
+| nodeTemplate                          | -                          | -                 | -               |
 | static                                | -                          | -                 | +               |
-| disruptions                           | -                          | -                 | -               |
-| cloudInstances.classReference         | -                          | +                 | -               |
+| update.maxConcurrent                  | -                          | +                 | -               |
```
Comment (yalosev) [inline modules/040-node-manager/docs/FAQ.md:365]:
```suggestion
| update.maxConcurrent                  | -                          | -                 | -               |
```

**Review by yalosev — CHANGES_REQUESTED** [review]

**Review by yalosev — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [3433](https://github.com/deckhouse/deckhouse/pull/3433) to the branch [release-1.42](https://github.com/deckhouse/deckhouse/tree/release-1.42) successful!

**Comment by deckhouse-BOaTswain** [comment]
Failure: cherry-pick commit e4fb36d9bab5f17422f5f5d667e08084c3eea0e4 to the branch [release-1.41](https://github.com/deckhouse/deckhouse/tree/release-1.41) failed. See [Job](https://github.com/deckhouse/deckhouse/actions/runs/3792923943) for details.

**Pull request merged**